app/src/features/react-hook-form-wrappers/DatePickerWrapped.tsx

```diff
@@ -0,0 +1,34 @@
+import type { DateInputProps } from "@navikt/ds-react";
+import { DatePicker, useDatepicker } from "@navikt/ds-react";
+import { useController } from "react-hook-form";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any -- usikker på hvordan rules skal types, og om det er såå viktig.
+type DatePickerWrappedProps = { name: string; rules?: any } & DateInputProps;
+
+export function DatePickerWrapped({
+  name,
+  rules = {},
+  ...dateInputProps
+}: DatePickerWrappedProps) {
+  const { field, fieldState } = useController({
+    name,
+    rules,
+  });
+  const datePickerProperties = useDatepicker({
+    onDateChange: async (date) => {
+      field.onChange(date);
+    },
```

Comment:
Småflikk, men her kan du bare sende inn metoden:
```suggestion
    onDateChange: field.onChange,
```